community/modules/scripts/htcondor-install/files/install-htcondor.yaml

```diff
@@ -47,26 +47,16 @@
   - name: Enable HTCondor Feature Release repository
     ansible.builtin.yum_repository:
       name: htcondor-feature
-      description: HTCondor Feature Releases (10.x.0)
+      description: HTCondor LTS Release (23.0)
       file: htcondor
-      baseurl: https://research.cs.wisc.edu/htcondor/repo/10.x/el$releasever/$basearch/release
+      baseurl: https://research.cs.wisc.edu/htcondor/repo/23.0/el$releasever/$basearch/release
       gpgkey: "{{ htcondor_key }}"
       gpgcheck: true
       repo_gpgcheck: true
       priority: "90"
-  - name: Enable HTCondor Feature Release Updates repository
-    ansible.builtin.yum_repository:
-      name: htcondor-feature-update
-      description: HTCondor Feature Release Updates (10.x.y)
-      file: htcondor
-      baseurl: https://research.cs.wisc.edu/htcondor/repo/10.x/el$releasever/$basearch/update
-      gpgkey: https://research.cs.wisc.edu/htcondor/repo/keys/HTCondor-10.x-Key
-      gpgcheck: true
-      repo_gpgcheck: true
-      priority: "90"
```

Comment:
I'm guessing there's just no version of this for 23.0?  Or is the repository just not needed anymore?

Reply:
Yes, they maintain the X.0 series differently from the X.Y series (which is similar to our "develop" branch). Repo not needed anymore.